libs/langchain-azure-cosmosdb/src/azure_cosmosdb_nosql.ts

```diff
@@ -195,7 +197,7 @@ export class AzureCosmosDBNoSQLVectorStore extends VectorStore {
         {
           path: "/vector",
           type: "quantizedFlat",
-        },
+        } as VectorIndex,
```

Comment:
Maybe `as const` works but not a huge deal

Reply:
Tried it but it still complained 😞